frontend/inverted-index.js

```diff
@@ -0,0 +1,126 @@
+/**
```

Comment:
adding `/*eslint-disable no-unused-vars*/` will fix that houndCI issue
http://eslint.org/docs/user-guide/configuring